src/core_modules/capture-core/components/WidgetEnrollmentEventNew/Validated/validated.actions.js

```diff
@@ -66,3 +67,5 @@ export const saveEvent = (serverData: Object, onSaveSuccessActionType?: string,
             rollback: onSaveErrorActionType && { type: onSaveErrorActionType, meta: { serverData, uid } },
         },
     });
+export const startCreateNewAfterCompleting = ({ enrollmentId, isCreateNew, orgUnitId, programId, teiId, availableProgramStage }: Object) =>
```

Comment:
Hey,
Based on the [parameters](https://github.com/dhis2/capture-app/pull/3065/files#diff-219843c9418d6aab189726c11f8623961fa99a504e1a74a0cae067f004169d1cR98) used to call the action, I believe `availableProgramStage` has a small typo and is missing `s` at the end. 
Should it be `availableProgramStages` instead?
Thanks!